src/zc/FileStorage/tests.py

```diff
@@ -522,18 +529,28 @@ def snapshot_in_time():
      'data2010-3-9T20:29:0.fs', 'data2010-3-9T20:29:0.fs.index',
      'snapshot.fs', 'snapshot.fs.index', 'snapshot.fs.lock', 'snapshot.fs.tmp']
 
-    >>> open('data2010-3-9T20:29:0.fs').read() == open('snapshot.fs').read()
+    >>> with open('data2010-3-9T20:29:0.fs', 'rb') as fd1, open('snapshot.fs', 'rb') as fd2:
+    ...     fd1.read() == fd2.read()
     True
 
     """
 
 
 def hexer(data):
-    return (data[:2] == ".h") and data or (".h" + data.encode("hex"))
+    if data[:2] == ".h":
+        return data
+
+    return ".h" + data.encode("hex")
 
 
 def unhexer(data):
-    return data and (data[:2] == ".h" and data[2:].decode("hex") or data)
+    if not data:
+        return None
```

Comment:
Maybe `return data` would be safer?  Consider `unhexer("")`: old code would return `""`, your code would return `None`.

Reply:
Good catch !